keyboards/claw44/keymaps/yfuku/keymap.c

```diff
@@ -0,0 +1,226 @@
+
+// alt_t
+#define KC_A_D ALT_T(KC_D)
+#define KC_A_K ALT_T(KC_K)
+#define KC_A_Z ALT_T(KC_Z)
+#define KC_A_SL ALT_T(KC_SLSH)
+#define KC_A_DEL ALT_T(KC_DEL)
+
+// cmd+shift_t
+#define KC_MS_Q SCMD_T(KC_Q)
+#define KC_MS_A SCMD_T(KC_A)
+#define KC_MS_S SCMD_T(KC_S)
+#define KC_MS_SC SCMD_T(KC_SCLN)
+#define KC_MS_ESC SCMD_T(KC_ESC)
+
+//
+#define KC_MR RCMD(KC_R)
+#define KC_MF RCMD(KC_F)
+#define KC_MW RCMD(KC_W)
+#define KC_MX RCMD(KC_X)
+#define KC_MC RCMD(KC_C)
+#define KC_MV RCMD(KC_V)
+#define KC_MTAB RCMD(KC_TAB)
+#define KC_MSF RCMD(RSFT(KC_F))
+#define KC_MSR RCMD(RSFT(KC_R))
+#define KC_MST RCMD(RSFT(KC_T))
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  // M_ = LCMD_T(
+  // A_ = ALT_T(
+  // C_ = LCTL_T(
+  // MS_ = SMD_T(
+  // R_ = LT(_RAISE
+  // L_ = LT(_LOWER
+
+  [_QWERTY] = LAYOUT_kc( \
+  //,----+----+----+----+----+----.     ,----+----+----+----+----+----.
+     ESC , Q  , W  , E  , R  , T  ,       Y  , U  , I  , O  , P  ,MINS,
+  //|----+----+----+----+----+----|     |----+----+----+----+----+----|
+    S_TAB, A  ,C_S , D  ,M_F , G  ,       H  ,M_J , K  ,C_L ,SCLN,S_ESC,
+  //|----+----+----+----+----+----+     |----+----+----+----+----+----|
+         , Z  , X  , C  , V  , B  ,       N  , M  ,COMM,DOT ,SLSH,    ,
+  //`----+----+----+----+----+----/     \----+----+----+----+----+----'
+             A_DEL,S_EN,L_SPC,C_BS,      C_BS,R_ENT,S_JA,A_DEL
+  //          `----+----+----+----'     `----+----+----+----'
+  ),
+
+  //   \ ^ ! & |  @ = + * % -
+  // ( # $ " ' ~  ← ↓ ↑ → ` )
+  //         { [  ] }
+
+  [_RAISE] = LAYOUT_kc( \
+  //,----+----+----+----+----+----.     ,----+----+----+----+----+----.
+         ,BSLS,CIRC,EXLM,AMPR,PIPE,      AT  ,EQL ,PLUS,ASTR,PERC,MINS,
+  //|----+----+----+----+----+----|     |----+----+----+----+----+----|
+     LPRN,HASH,DLR ,DQT ,QUOT,TILD,      LEFT,DOWN, UP ,RGHT,GRV ,RPRN,
+  //|----+----+----+----+----+----|     |----+----+----+----+----+----|
+        ,     ,    ,    ,LCBR,LBRC,      RBRC,RCBR,    ,    ,    ,    ,
+  //`----+----+----+----+----+----/     \----+----+----+----+----+----'
+                   ,    ,BSPC,    ,          ,    ,    ,RST
+  //          `----+----+----+----'     `----+----+----+----'
+  ),
+
+  [_LOWER] = LAYOUT_kc( \
+  //,----+----+----+----+----+----.     ,----+----+----+----+----+----.
+         ,    ,    ,MSF ,MSR ,MST ,          ,EQL ,PLUS,ASTR,PERC,MINS,
+  //|----+----+----+----+----+----|     |----+----+----+----+----+----|
+         , 1  , 2  , 3  , 4  , 5  ,       6  , 7  , 8  , 9  , 0  ,    ,
+  //|----+----+----+----+----+----|     |----+----+----+----+----+----|
+         ,    ,    ,    ,    ,    ,          ,    ,COMM,DOT ,SLSH,    ,
+  //`----+----+----+--+-+----+----/     \----+----+----+----+----+----'
+               RST ,    ,    ,    ,          ,DEL ,    ,
+  //          `----+----+----+----'     `----+----+----+----'
+  ),
+};
+
+void persistent_default_layer_set(uint16_t default_layer) {
```

Comment:
Could you remove this function?  

It's been depreciated in favor of the `set_single_persistent_default_layer` function (which is a core function).

https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer

Reply:
I removed that function.